### PR TITLE
Add python 3.4

### PR DESCRIPTION
**Pull request by spencerahill**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,9 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+
     - CONDA_PY=27
+    - CONDA_PY=34
```
Comment (ocefpaf) [inline .travis.yml]: Conda-forge does not support `Python 3.4` and manual edits to the feedstock is discourage.
The main reason is the load in the CIs as we cannot afford to build 3 version for Python 3 (3.4, 3.5, and 3.6).

Please consider re-rendering this with latest conda-smithy to remove the addition matrix items. (Note that the package you created with this PR will remain in the channel, so it is a win-win :wink:)

Comment (spencerahill) [inline .travis.yml]: @ocefpaf sorry about that.  I saw at least one other package (xarray) that has 2.7 and 3.4-3.6 channels all on conda-forge, so I thought it was OK.  Are they a special case?

Comment (ocefpaf) [inline .travis.yml]: > Are they a special case?

Nope. Probably just pending a re-render.

Comment (ocefpaf) [inline .travis.yml]: BTW 2.7 is still maintained. We dropped only 3.4.

Comment (spencerahill) [inline .travis.yml]: Ok, thanks for the fast response.  Will revert the changes from this PR.

Comment (ocefpaf) [inline .travis.yml]: Are you familiar with [conda-smithy](https://github.com/conda-forge/staged-recipes/wiki/conda-smithy-rerender)?

I recommend you to re-render using that instead of changing the feedstock boilerplate configuration manually.

Comment (spencerahill) [inline .travis.yml]: Ah, good call, will use that.  Thanks for your help -- sorry for my mistakes.

Comment (ocefpaf) [inline .travis.yml]: > Thanks for your help -- sorry for my mistakes.

No problem. Feel free to bug me when in doubt.
